content/kubermatic/main/installation/install-kkp-CE/add-seed-cluster-CE/_index.en.md

```diff
@@ -92,42 +160,119 @@ cluster backups.
 
 ```bash
-kubectl replace -f charts/kubermatic-operator/crd/
+helm --namespace minio upgrade --install --wait --values /path/to/your/helm-values.yaml minio charts/minio/
+helm --namespace kube-system upgrade --install --wait --values /path/to/your/helm-values.yaml s3-exporter charts/s3-exporter/
+```
+
+You will also need to manually ensure that a suitable `StorageClass` called `kubermatic-fast` exists.
+
+### Option 3: No Installation
+
+If you have manually ensured that a suitable `StorageClass` called `kubermatic-fast` exists (see [Create a StorageClass](#create-a-storageclass))
+and do not want to install MinIO, no installation step is needed here. Everything else will be set up by the Kubermatic Operator once
+the `Seed` resource has been created.
+
+### Set Up MinIO Bucket (only needed with MinIO)
+
+If you are using MinIO, a bucket needs to be created for cluster backups to be stored in. This can be done for example
+via a `Job` resource that spawns a `Pod` running the `mc` command against the freshly deployed MinIO service. Below you
+find an example `Job` definition. If you want to change the bucket name, replace `src/kkpbackup` with `src/YOUR_BUCKET_NAME`
+in the `args` part of the template.
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-minio-backup-bucket
+  namespace: minio
+spec:
+  backoffLimit: 2
+  template:
+    spec:
+      containers:
+        - name: mc
+          image: quay.io/kubermatic/util:2.2.0
+          args:
+            - /bin/sh
+            - -c
+            - mc --insecure config host add src http://minio.minio.svc.cluster.local:9000 "$MINIO_ACCESS_KEY" "$MINIO_SECRET_KEY" && mc --insecure mb src/kkpbackup
+          env:
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: minio
+                  key: accessKey
+            - name: MINIO_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: minio
+                  key: secretKey
+          securityContext:
+            runAsNonRoot: false
+            runAsUser: 0
+      restartPolicy: Never
 ```
 
-After configuring the required options, you can install the charts:
+Put this into a file (e.g. called `minio-bucket-job.yaml`) and create it via:
 
 ```bash
-helm --namespace minio upgrade --install --wait --values /path/to/your/helm-values.yaml minio charts/minio/
-helm --namespace kube-system upgrade --install --wait --values /path/to/your/helm-values.yaml s3-exporter charts/s3-exporter/
+kubectl create -f ./minio-bucket-job.yaml
+#Job/create-minio-backup-bucket created
 ```
 
-## Add the Seed Resource
+Supervise the `Job` to ensure the bucket gets created successfully.
+
+## Add Seed Resource
+
+Next you need to prepare and apply the `Seed` resource that will connect the master cluster to the new seed cluster to finalize
+the seed setup (applying CRDs and creating seed-specific workloads).
+
+### Create Seed Kubeconfig
```

Comment:
We should clarify if you need to follow this if you have shared master/seed setup.